drivers/adc/adc_stm32.c

```diff
@@ -229,6 +229,8 @@ static int adc_stm32_dma_start(const struct device *dev,
 	}
 #elif defined(ADC_VER_V5_X)
 	LL_ADC_REG_SetDataTransferMode(adc, LL_ADC_REG_DMA_TRANSFER_LIMITED);
+#else
+	LL_ADC_REG_SetDMATransfer(adc, LL_ADC_REG_DMA_TRANSFER_LIMITED);
```

Comment:
do you think this can potentially break any other mcu line?
in case we want to make a PR to the Zephyr repo

Reply:
I don't know for sure but I don't expect it. I could not find any documentation what "ADC_VER_V5_V90" and "ADC_VER_V5_X" mean, so maybe the guy who wrote that code knows more about the internals of the STM32. Probably this code was not tested on many other mcu lines since it is quite new and this part of the code is only used if you enable the DMA mode via CONFIG_ADC_STM32_DMA.